modules/redirects.ts

```diff
@@ -0,0 +1,48 @@
+import { defineNuxtModule, extendRouteRules } from '@nuxt/kit';
+import { Directus } from '@directus/sdk';
+
+export type Redirect = {
+	url_old: string;
+	url_new: string;
+	response_code: string | number;
+}
+
+export default defineNuxtModule({
+	  async setup(moduleOptions, nuxt) {
+	    const directusUrl = nuxt.options.runtimeConfig.public.directusUrl
+	    const directusToken = nuxt.options.runtimeConfig.directusToken
+	    if (!directusUrl) throw new Error('Missing directusUrl in runtimeConfig')
+	    if (!directusToken)
+	      throw new Error('Missing directusToken in runtimeConfig')
```

Comment:
Wrap in {} as per eslint rules

```suggestion
	    if (!directusToken) {
	      throw new Error('Missing directusToken in runtimeConfig');
	    }
```